css/src/core/dismiss.scss

```diff
@@ -0,0 +1,9 @@
+@mixin dismiss {
```

Comment:
should this be in /mixins? Due to import order, I think we want to ensure it's available early in the order.

Reply:
Updated! Forget to move it after realized creating mixin is a good idea :D